working/0546-patterns/patterns-feature-specification.md

```diff
@@ -1885,14 +1900,27 @@ Where `<keyword>` is `var` or `final` is treated like so:
     }
     ```
 
-#### Pattern-if statement
+#### If-case statement
 
 1.  Evaluate the `expression` producing `v`.
 
-2.  Match the `pattern` against `v`.
+2.  Match the `pattern` in the `caseHead` against `v`.
+
+3.  If the match succeeds:
```

Comment:
Worth mentioning that new variables are available here?

Reply:
I think the "Variables and scope" section covers this pretty well, but if you think it's better to explain it here I could do that.

Reply:
\*\* goes to read "Variables and scope"\*\*